frontend/src/components/PairRoom/PairRoomOnboardingModal/Progress/Progress.tsx

```diff
@@ -0,0 +1,29 @@
+import Button from '@/components/common/Button/Button';
+import { Step } from '@/components/PairRoom/PairRoomOnboardingModal/PairRoomOnboardingModal.type';
+
+import * as S from './Progress.styles';
+
+interface ProgressProps {
+  step: Step;
+  isRoleSelected: boolean;
+}
+
+const Progress = ({ step, isRoleSelected }: ProgressProps) => (
```

Comment:
[사소한 네이밍 변경 제안] - `Progress` 는 그냥 `진행` 이라는 의미라서 컴포넌트 네이밍으로는 어색한 느낌이에요!
```suggestion
const ProgressBar = ({ step, isRoleSelected }: ProgressBarProps) => (
```

Reply:
수정했습니다! 훨씬 더 명확해졌네요! 👍